3b_parameters/MODIS_MCD12Q1_V6/1_download/download_modis_mcd12q1_v6.ipynb

```diff
@@ -24,7 +24,9 @@
     "from netrc import netrc\n",
     "from pathlib import Path\n",
     "from shutil import copyfile\n",
-    "from datetime import datetime"
+    "from datetime import datetime\n",
+    "import concurrent\n",
```

Comment:
Can you update `requirements.txt` with the version of concurrent?

Reply:
This is path of the standard library, not needed for requirements.txt